### PR TITLE
Pass events to Dispatcher callbacks.

### PR DESCRIPTION
**Pull request by jtlan**

### CODE REVIEW

File: src/dispatchers/mouseDispatcher.ts [hunk 1]
```diff
@@ -130,7 +130,7 @@ export module Dispatcher {
       var newMousePosition = this._computeMousePosition(e.clientX, e.clientY);
       if (newMousePosition != null) {
         this._lastMousePosition = newMousePosition;
-        b.broadcast();
+        b.broadcast(this._lastMousePosition, e);
```
Comment (bluong) [inline src/dispatchers/mouseDispatcher.ts]: I remember seeing up above that there was a getter.  Can we use the getter here instead?

File: src/dispatchers/mouseDispatcher.ts [hunk 2]
```diff
@@ -71,7 +71,7 @@ export module Dispatcher {
     }
 
     protected _getWrappedCallback(callback: Function): Core.BroadcasterCallback<Dispatcher.Mouse> {
-      return () => callback(this.getLastMousePosition());
+      return (md: Dispatcher.Mouse, p: Point, e: MouseEvent) => callback(p, e);
```
Comment (bluong) [inline src/dispatchers/mouseDispatcher.ts]: Where did this point argument come from?

Comment (jtlan) [inline src/dispatchers/mouseDispatcher.ts]: The `Broadcaster` passes it.

File: test/dispatchers/dispatcherTests.ts [hunk 3]
```diff
@@ -115,11 +115,11 @@ describe("Dispatchers", () => {
       var md = Plottable.Dispatcher.Mouse.getDispatcher(<SVGElement> target.node());
 
       var cb1Called = false;
-      var cb1 = function(p: Plottable.Point) {
+      var cb1 = function(p: Plottable.Point, e: MouseEvent) {
         cb1Called = true;
       };
       var cb2Called = false;
-      var cb2 = function(p: Plottable.Point) {
```
Comment (bluong) [inline test/dispatchers/dispatcherTests.ts]: You can make the callbacks
var cb1 = () => cb1Called = true;
and the same for cb2

**Comment by bluong** [comment]
Minor test issues

**Comment by crmorford** [comment]
Is Travis still flaking out?

**Comment by crmorford** [comment]
Regression pass?

**Comment by jtlan** [comment]
@crmorford Yeah, just a regression pass. Quick check of `Interaction.Key` and `Interaction.Hover` should be sufficient.

**Pull request merged**